### DISCUSSION TITLE
Merging different EBSD measurement to have an Stitching IPF Map

### DISCUSSION THREAD

**Discussion by nargesmbg** [mtex-toolbox/mtex#1826] [Ask Anything]
hi everyone, 

I did 5 ebsd measurement on sequential location on my sample to cover a large area and having representative measurement results. 
I'm struggling with combining and merging my data set, according to the spatial coordnation to make an stitched EBSD IPF map. 
is there any way to do so? 

Best 
Narges

**Comment by kilir** [comment]
Hi,
basically you "just" need to change the x,y coordinates of 4 of those maps, such that they are placed at the proper position (assuming they all have the same phases in the same order) and then just concatenate ( `[ebsd1 ebsd2 ebsd3 ...]` ) them.
Cheers,
Rüdiger

**Comment by nargesmbg** [comment]
hi Rüdiger, 
could you please explain to me how can I change the x and y coordinates of maps? 
Best 
Narges

**Reply by kilir** [reply]
Hi,
easiest would be to use the `EBSD/shift` or `EBSD/affinetrans` functions.
Cheers, 
Rüdiger

**Comment by AzdiarGazder** [comment]
Hi @nargesmbg ,

To help you stitch your maps into one, please refer to the "stitch" function in the following weblink: https://github.com/AzdiarGazder/mtexTools.

Warm regards,
Azdi
